### PR TITLE
set up project

### PR DESCRIPTION
**Pull request by josh-chou**

### CODE REVIEW

File: ban/__init__.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            if "BAN" not in req_json:
+                raise ValueError("Missing BAN parameter")
+
+            # query socrata
+            params["BAN"] = req_json.get("BAN")
+            response = requests.get(
+                os.getenv('BAN_API_URL'),
+                headers={
+                    'Authorization': 'Basic ' + os.getenv('ADDRESS_SVC_AUTH_API_KEY'),
+                    'X-App-Token': os.getenv('ADDRESS_SVC_APP_TOKEN')
```
Comment (hshaosf) [inline ban/__init__.py]: Probably rename `ADDRESS_SVC_AUTH_API_KEY` to something like `BAN_API_AUTH_API_KEY`, and similarly `ADDRESS_SVC_APP_TOKEN` to `BAN_API_APP_TOKEN`

File: ban/__init__.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    "DBAName": "dbaname",
+    "StreetAddress": "streetaddress",
+    "City": "city",
+    "State": "state",
+    "PostalCd": "postalcd",
+    "LIN": "lin",
+    "BusstartDate": "busstartdate",
+    "BusEndDate": "busenddate",
+    "LocstartDate": "locstartdate",
+    "LocEndDate": "locenddate",
```
Comment (hshaosf) [inline ban/__init__.py:27]: Comparing the outputs with the existing API, we probably need to some light date conversion to matches with existing MM-DD-YYYY format.

File: ban/__init__.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    "MailingAddress": "mailingaddress",
+    "MailCityStateZip": "mailcitystatezip",
+    "LocationNumber": "locationnumber",
+    "OrgType": "orgtype",
+    "MailZip": "postalcd",
+    "MailState": "state",
+    "MailCity": "city"
```
Comment (hshaosf) [inline ban/__init__.py]: Based on what we discussed, I guess let's remove these since they are not in the dataset.

**Review by hshaosf — APPROVED** [review]
LGTM! 👍 

just some minor nit-picky items :)

File: ban/__init__.py (hunk 3 above)
Comment (josh-chou) [reply ban/__init__.py]: b25302f

File: ban/__init__.py (hunk 1 above)
Comment (josh-chou) [reply ban/__init__.py]: 7ab99d1

File: ban/__init__.py (hunk 2 above)
Comment (josh-chou) [reply ban/__init__.py:27]: c875181

**Pull request merged**